### ISSUE TITLE
My page is shown in Dark Mode by default

### ISSUE THREAD

**Issue by estebanhirzfeld** [JoseSoteloCohen/Blackout-WordPress-Dark-mode-Widget#20]
My page is shown in Dark Mode by default

**Comment by JoseSoteloCohen** [comment]
Hello!

It must be because you have "Want to match the OS mode?" checkbox enabled.

Check it and let me know.

**Comment by phillipjohnston** [comment]
Hello, 

I also reported this on the plugin support page. Sorry for the duplicate. But this happens on my website without the "Want to match the OS mode?" checkbox enabled. The pages follow the setting of the OS no matter what, unless I manually toggle the setting.

This happens for me with Safari on iOS and OS X, even in private mode with no previous cookies set.

**Comment by JoseSoteloCohen** [comment]
@phillipjohnston Hello! that shouldn't be happening.

I'll check if there are known bugs from the script side or if it's something from the WordPress implementation.

Thank you very much for letting me know and giving me the chance to fix it.

P.D.: I never get support emails from WordPress, I don't know why, so it's good that you reached out here.

**Comment by phillipjohnston** [comment]
Let me see if I can get my staging site set up as an example.

**Comment by phillipjohnston** [comment]
Here you go - https://02777198f8.nxcli.net/. If I toggle the OS light/dark mode and reload the page, the light/dark mode setting changes.

here are the settings used in the plugin.

![Screen Shot 2020-08-05 at 15 05 03](https://user-images.githubusercontent.com/1709063/89468912-4c30fb00-d72d-11ea-95da-f3ffcfaa04e2.png)

**Comment by phillipjohnston** [comment]
Tested in Firefox as well - same thing happens there.

**Comment by phillipjohnston** [comment]
Ok, I did some poking around in the blackout.php file. I think I figured out what's going on. But take this with a grain of salt and make the proper fix - I'm a C and C++ guy, and don't know much PHP.

In the developer output in the browser, I noticed that the JS file default uses a number for truth: !0.

I changed this snippet to use integers instead of 'true' and 'false':

```
    if ($blackout_options['blackout_match_os'] == 1){
        $blackout_match_os = 1;
    }else{
        $blackout_match_os = 0;
    }
```

And removed the single quotes around the variable:

```
autoMatchOsTheme: {$blackout_match_os}, // default: true
```

This made the option work as expected, and now I can toggle it correctly in the plugin settings using the check box.

I think the same thing needs to happen for the cookie setting as well.
